### PR TITLE
Change sentry replays only to log YPP page actions

### PR DESCRIPTION
**Pull request by attemka**

### CODE REVIEW

File: packages/atlas/src/MainLayout.tsx [hunk 1]
```diff
@@ -53,6 +54,20 @@ export const MainLayout: FC = () => {
     onExitClick: () => closeDialog(),
   })
 
+  useEffect(() => {
+    const stopReplay = async () => await SentryLogger.replay?.stop()
+
+    if (location.pathname === absoluteRoutes.viewer.ypp()) {
```
Comment (drillprop) [inline packages/atlas/src/MainLayout.tsx:64]: why can't we just move this to ypp view?

File: packages/atlas/src/MainLayout.tsx [hunk 2]
```diff
@@ -53,6 +54,20 @@ export const MainLayout: FC = () => {
     onExitClick: () => closeDialog(),
   })
 
+  useEffect(() => {
```
Comment (drillprop) [inline packages/atlas/src/MainLayout.tsx:58]: We should return when the sentry is not set in atlas.config.yml

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/MainLayout.tsx (hunk 1 above)
Comment (attemka) [reply packages/atlas/src/MainLayout.tsx:64]: this was the initial Idea, but we have async stopReplay, which would be tricky to execute on component unmount

File: packages/atlas/src/MainLayout.tsx (hunk 2 above)
Comment (attemka) [reply packages/atlas/src/MainLayout.tsx:58]: good catch, added this.

**Review by drillprop — APPROVED** [review]

**Pull request merged**